usr/src/lib/libfakekernel/common/kmisc.c

```diff
@@ -69,6 +117,32 @@ delay(clock_t ticks)
 	(void) poll(0, 0, msec);
 }
 
+int
+issig(int why)
+{
+	return (0);
+}
+
+/* ARGSUSED */
```

Comment:
Here too, how about a cyclic.c?

Would rather not see kmisc.c turn into more of a "catch" all...

Reply:
Sure.  This sounds good to me.